<ns0:entry xmlns:ns0="http://www.w3.org/2005/Atom" xmlns:ns1="http://schemas.google.com/g/2005" xmlns:ns2="http://schemas.google.com/projecthosting/issues/2009" ns1:etag="W/&quot;DkIHSX47eCl7ImA9WhBTGEo.&quot;">
		<ns0:id>http://code.google.com/feeds/issues/p/nativeclient/issues/3087/comments/full/13</ns0:id><ns0:author>
			<ns0:name>bugdro...@chromium.org</ns0:name><ns0:uri>/u/101788990881243245974/</ns0:uri></ns0:author><ns0:content type="html">The following revision refers to this bug:
    http://src.chromium.org/viewvc/native_client?view=rev&amp;revision=10794

------------------------------------------------------------------------
r10794 | mark@chromium.org | 2013-02-14T15:14:00.443074Z

Changed paths:
   M http://src.chromium.org/viewvc/native_client/trunk/src/native_client/tests/exception_test/exception_crash_test.c?r1=10794&amp;r2=10793&amp;pathrev=10794
   M http://src.chromium.org/viewvc/native_client/trunk/src/native_client/tests/exception_test/nacl.scons?r1=10794&amp;r2=10793&amp;pathrev=10794

Switch exception_test_crash_in_syscall to test_crash(NACL_TEST_CRASH_MEMORY).

Similar to r10789, but for the exception handling test, crash-in-syscall can
use the dedicated test_crash syscall instead of imc_recvmsg for more reliable
results.

BUG=https://code.google.com/p/nativeclient/issues/detail?id=3087
TEST=scons run_exception_test_crash_in_syscall
Review URL: https://codereview.chromium.org/12207164
------------------------------------------------------------------------</ns0:content><ns0:updated>2013-02-14T20:42:18.000Z</ns0:updated><ns0:published>2013-02-14T20:42:18.000Z</ns0:published><ns2:updates /><ns0:title>Comment 13 by bugdro...@chromium.org</ns0:title><ns0:link href="http://code.google.com/p/nativeclient/issues/detail?id=3087#c13" rel="alternate" type="text/html" /><ns0:link href="https://code.google.com/feeds/issues/p/nativeclient/issues/3087/comments/full/13" rel="self" type="application/atom+xml" /></ns0:entry>